script/docker/snapcraft.Dockerfile

```diff
@@ -7,11 +7,19 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
 RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
 
 # add custom tools required for electron-builder
-RUN apt -qq update
-RUN apt -qq install --yes \
-  nodejs \
-  yarn \
-  # needed for pacman builds
-  bsdtar \
-  # needed for RPM builds
-  rpm
+RUN apt-get -qq \
+    # yarn's public key is currently expired, so we have to hack around it
+    -o Acquire::AllowInsecureRepositories=true \
```

Comment:
Because without this `apt-get update` errors because of this expired key:

```
Step 6/7 : RUN apt-get -qq     -o Acquire::AllowInsecureRepositories=true     -o Acquire::AllowDowngradeToInsecureRepositories=true     update
 ---> Running in 4b1e88ebe4b9
W: GPG error: https://dl.yarnpkg.com/debian stable InRelease: The following signatures were invalid: KEYEXPIRED 1507181400  KEYEXPIRED 1546376218  KEYEXPIRED 1546372003  KEYEXPIRED 1546372003  KEYEXPIRED 1507181400  KEYEXPIRED 1546376218  KEYEXPIRED 1546372003  KEYEXPIRED 1507181400  KEYEXPIRED 1546376218  KEYEXPIRED 1546372003
W: The repository 'https://dl.yarnpkg.com/debian stable InRelease' is not signed.
```